client/lib/post-types-list/list.js

```diff
@@ -73,7 +75,7 @@ PostTypesList.prototype.fetch = function( siteId ) {
  * @return {array} services
  **/
 PostTypesList.prototype.parse = function( data ) {
-	return data.post_types;
+	return values( omit( data, '_headers' ) );
```

Comment:
I do think we should follow the patterns in `wpcom-proxy-request` and `wpcom-xhr-request`, in this particular example as @nylen said, this data is being added in by the `rest-proxy` handler.